src/styles/base/_forms.scss

```diff
@@ -13,6 +13,8 @@ select {
   font-size: $base-font-size;
   height: $forms-height;
   border: $base-border;
+  min-width: rem(180);
```

Comment:
Maybe also putting this into a variable, just in case we need to use in other elements